src/render/repl.rs

```diff
@@ -95,7 +95,11 @@ fn repl_render_stream_inner(
                     writer.flush()?;
                 }
                 ReplyStreamEvent::Done => {
-                    queue!(writer, style::Print("\n"))?;
+                    #[cfg(target_os = "windows")]
+                    let eol = "\n\n";
```

Comment:
Line markers for windows are `\r\n`

Reply:
It's just in the repl and does not need to follow the specifications of windows newline

`\r\n` does not work, must be `\n\n`.

This is weird. Maybe there is some issue with corssterm.